code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1264,3 +1264,12 @@
 			Stun(3)
 	..()
 
+/mob/living/silicon/robot/revive(full_heal = 0, admin_revive = 0)
+	if(..()) //successfully ressuscitated from death
+		if(camera && !wires.is_cut(WIRE_CAMERA))
+			camera.toggle_cam(src,0)

```

Comment:
borg camera is always turned off when it dies. Reviving the borg turn the camera on again, unless the camera's wires have been cut.